docs/reference/commandline/secret_create.md

```diff
@@ -28,7 +28,7 @@ Options:
 ## Description
 
 Creates a secret using standard input or from a file for the secret content. You must run this
-command on a manager node.
+command on a manager node. The content of secret must be less than 500 KB.
```

Comment:
I'm not sure it should be part of the first paragraph; I think having some paragraphs after this that describe what secrets are could work though. Perhaps we should either _copy_  (a shortened version of) the "about secrets" section; https://docs.docker.com/engine/swarm/secrets/#about-secrets 

At least we need to add a link to the page (https://docs.docker.com/engine/swarm/secrets/) somewhere in this reference doc